keras_retinanet/utils/config.py

```diff
@@ -24,6 +25,14 @@ def read_config_file(config_path):
     config = configparser.ConfigParser()
     config.read(config_path)
 
+    assert os.path.isfile(config_path), "Could not find {}.".format(config_path)
```

Comment:
This check is now after attempting to read the file. I would much prefer if you switch to something like this:
```python
with open(config_path, 'r') as file:
    config.read_file(file)
```

That will throw a regular `FileNotFoundError`, which makes it easier to catch the error if desired.